core/domain/email_manager.py

```diff
@@ -934,3 +938,92 @@ def send_test_email_for_bulk_emails(tester_id, email_subject, email_body):
     _send_email(
         tester_id, tester_id, feconf.BULK_EMAIL_INTENT_TEST,
         email_subject, email_body, tester_email, sender_name=tester_name)
+
+
+def send_mail_to_onboard_new_reviewers(user_id, category):
+    """Sends an email to users who have crossed the threshold score.
+
+    Args:
+        user_id: str. The ID of the user who is being offered to become a
+            reviewer.
+        category: str. The category that the user is being offered to review.
+    """
+
+    email_subject = 'Invitation to review suggestions'
+
+    email_body_template = (
```

Comment:
I think this could use some copywriting. Could you make a doc with the text here and send it to me, please?

Reply:
Here is the doc https://docs.google.com/document/d/12Mvxjy92Yxq4VjCzl6OcnGKWwDATZ51oTkQBGVIK6No/edit#